auslib/db.py

```diff
@@ -1745,6 +1745,23 @@ def hasPermission(self, username, thing, action, product=None, transaction=None)
         return True
 
 
+class Dockerflow(AUSTable):
+    def getWatchdogValue(self, transaction=None):
+        return self.select(transaction=transaction)
+
+    def incrementWatchdogValue(self, changed_by, transaction=None, dryrun=False):
+        if not self.db.hasPermission(changed_by, "dockerflow", "modify", transaction=transaction):
+            raise PermissionDeniedError("%s is not allowed to modify dockerflow" % changed_by)
+
+        watchdog = self.getWatchdogValue()
```

Comment:
It's totally fine to keep it, but it's not strictly necessary to have a getter here. AUSTable subclasses are allowed to invoke self.select directly.